internal/gui/gui_unix.go

```diff
@@ -3,8 +3,23 @@
 
 package gui
 
-import "github.com/skycoin/skywire/pkg/util/osutil"
+import (
+	"os/user"
+
+	"github.com/skycoin/skywire/pkg/util/osutil"
+)
 
 func platformExecUninstall() error {
 	return osutil.Run("/bin/bash", "-c", deinstallerPath)
 }
+
+func checkRoot() bool {
+	thisUser, err := user.Current()
+	if err != nil {
+		panic(err)
```

Comment:
I would prefer to not panic here and log a fatal err with some context.

Reply:
the logging overhead is unnecessary and it doesn't really need to fail, only to return true if root is detected.

this check is for limiting the available  buttons in the systray to ones that are appropriate to launch by the current user. The browser won't be allowed to run as root in the userspace by the system, typically.